course/layouts/partials/download_page.html

```diff
@@ -0,0 +1,51 @@
+<div class="download-page container">
+    {{- if .Params.description -}}
+        <div class="row">
+            <div class="label col-3">Description:</div>
+            <div class="col-9">{{ .Params.description }}</div>
+        </div>
+    {{- end -}}
+    {{- if .Params.resourcetype -}}
+        <div class="row">
+            <div class="label col-3">Resource Type:</div>
+            <div class="col-9">{{ .Params.resourcetype }}</div>
+        </div>
+    {{- end -}}
+    {{- if .Params.file_type -}}
+        <div class="row">
+            <div class="label col-3">File Type:</div>
+            <div class="col-9">{{ .Params.file_type }}</div>
+        </div>
+    {{- end -}}
+    {{- $instructors := partial "get_instructors.html" $.Site.Data.course.instructors.content -}}
+    {{- if $instructors -}}
+        <div class="row">
+            <div class="label col-3">Instructor(s):</div>
+            <div class="col-9">
+                {{ range $instructors }}
+                    <a href="{{ .url }}">{{ .title }}</a>
+                {{ end }}
+            </div>
+        </div>
+    {{- end -}}
```

Comment:
we decided to remove instructors because the same info is in the course info drawer.